Windows/resource.h

```diff
@@ -1,91 +1,1338 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
 // Used by ppsspp.rc
 //
-
+#define SW_HIDE                         0
```

Comment:
It seems weird that SW_HIDE is now defined here?

-[Unknown]

Reply:
I think this is because of Visual Studio